symphony/content/content.publish.php

```diff
@@ -1668,7 +1668,7 @@ private function prepareAssociationsDrawer($section)
                         // get associated entries if entry exists,
                         if ($entry_id) {
                             $relation_field = FieldManager::fetch($as['child_section_field_id']);
-                            $entry_ids = $relation_field->findParentRelatedEntries($as['parent_section_field_id'], $entry_id);
+                            $entry_ids = $relation_field->findParentRelatedEntries($as['child_section_field_id'], $entry_id);
```

Comment:
>  isn't this more of an issue within the respective fields then?

Yes and no, [because of this](https://github.com/symphonycms/symphony-2/blob/5763872ba7bb383995d94fd7e1d34f76c206f8ae/symphony/lib/toolkit/class.field.php#L1906)